internal/syscallfs/syscallfs.go

```diff
@@ -14,15 +11,20 @@ import (
 //
 // See https://github.com/golang/go/issues/45757
 type FS interface {
-	fs.FS
+	// Open is only defined to match the signature of fs.FS until we remove it.
+	// Once we are done bridging, we will remove this function. Meanwhile,
+	// using it will panic to ensure internal code doesn't depend on it.
+	Open(name string) (fs.File, error)
 
 	// OpenFile is similar to os.OpenFile, except the path is relative to this
 	// file system.
 	OpenFile(name string, flag int, perm fs.FileMode) (fs.File, error)
+	// ^^ TODO: switch to syscall.X, not os.X
 
 	// Mkdir is similar to os.Mkdir, except the path is relative to this file
 	// system.
 	Mkdir(name string, perm fs.FileMode) error
+	// ^^ TODO: switch to syscall.X, not os.X
```

Comment:
what's os.X?

Reply:
os.Mkdir -> syscall.Mkdir except doing this we need to maybe end up mapping some syscall flags which go does for us when we use os. stuff

Reply:
made a better comment